week 6.cpp

```diff
@@ -0,0 +1,204 @@
+#include <iostream>
+#include <stdlib.h>
+using namespace std;
+int id=0;
+class Item
+{
+	int item_id;
+	//char item_name;
+	float price;
```

Comment:
You have to be careful with float as you may run into rounding issues when dealing with money.  The classic example is that .1 can't be stored as a floating point number:
http://www.exploringbinary.com/why-0-point-1-does-not-exist-in-floating-point/